unrootfs.sh

```diff
@@ -114,4 +101,25 @@ if [ -n "$BL_DEV" -a -r "${ROOT_DIR}/etc/default/u-boot-device" ]; then
     "${ROOT_DIR}/usr/bin/install-u-boot" "${BL_DEV}" "${ROOT_DIR}"
 fi
 
+if [ -n "$BL_DEV" -a -r "${ROOT_DIR}/etc/default/systemd-boot" ]; then
+    [ -d /dev/disk/by-uuid ] || die "/dev/disk/by-uuid not found, gen-systemd-boot would be misled"
+    mount_pseudo
+    # NOTE(yoctozepto): aarch64 requires console= config, else it hangs (as of 20240731 on EC2 t4g)
+    # TODO(yoctozepto): aarch64 rootfs already has the initramfs
```

Comment:
@q66 Despite that I wrote just about aarch64, now also x86_64 gets the initramfs before this stage. It gets created whenever linux-lts is installed due to initramfs hooks. It was not the case even though the history suggests the same hook was installed (was it blocked from running?). Anyhow, that effort is partially wasted because the initramfs generator has no idea about the UUID of the block device for resume operations - it is going to miss it. Thus, I could skip installing linux-lts at rootfs level and postpone it till now (then get just one, proper build) - this is a bit ugly as it splits the pkg installation effort (also, as an alternative, the entire non-bootstrap part of pkgs could be installed on a loop device for sanity - although this introduces more complexity in the process) - or the previous behaviour could be restored - or else we simply decide to waste the original initramfs build and be fine with that (that's also an option). WDYT?